build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -297,95 +306,132 @@ def run_benchmarks_for_category(
   normal_benchmark_tool_path = adb_push_to_tmp_dir(normal_benchmark_tool,
                                                    relative_dir="normal-tools",
                                                    verbose=verbose)
+  benchmark_results_dir = os.path.join(tmp_dir, "benchmark-results")
```

Comment:
Some comments detailing the intent will make it easier to understand later, like "Create temporary directories on the host machine for benchmark results".

Applicable to the following new code too.